eclipse-collections-api/src/main/java/org/eclipse/collections/api/map/MutableMapIterable.java

```diff
@@ -152,6 +152,20 @@ default boolean removeIf(Predicate2<? super K, ? super V> predicate)
      */
     MutableMapIterable<K, V> withKeyValue(K key, V value);
 
+    /**
+     * This method allows mutable maps the ability to add elements of a Map to their existing
+     * elements. The original map is modified and then returned.
```

Comment:
The wording is confusing to me. I'd suggest something like "Copies all of the mappings from the specified map to this map and returns this map" or "Similar to putAll() but returns `this` instead of `void`"